nexus/reconfigurator/planning/src/system.rs

```diff
@@ -805,6 +879,8 @@ pub struct SledHwInventory<'a> {
     pub baseboard_id: &'a BaseboardId,
     pub sp: &'a nexus_types::inventory::ServiceProcessor,
     pub rot: &'a nexus_types::inventory::RotState,
+    pub stage0: Option<Arc<nexus_types::inventory::Caboose>>,
+    pub stage0_next: Option<Arc<nexus_types::inventory::Caboose>>,
```

Comment:
same q here I guess -- can we prefill these values to something reasonable? In production we always have these filled out, right?

Reply:
Same as above, I'd like to ask @davepacheco why it was done this way with the SPs to begin with